workers/httpsrv/httpsrv.go

```diff
@@ -82,6 +85,18 @@ func HealthHandler(now func() time.Time) http.HandlerFunc {
 
 }
 
+// CORSHandler handles pre-flight CORS requests.
+var CORSHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
What if I want different headers in this handler?
Wouldn't we be better served by a function which returns the cord handler, that also allows us to pass a list of desired headers to give to `Access-Control-Allow-Headers`?

Reply:
Yes, I figured that may be better. This was the simple version, configurable version to follow